### PR TITLE
SAN Copy Updates

### PR DESCRIPTION
**Pull request by cameron-eyds**
*Issue #:* /bcgov/entity#21555
*Description of changes:*
- Copy and UXA Updates for SAN Flow

*Issue #:* /bcgov/entity#21578
*Description of changes:*
- Payload filtering of DELETED sections

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the PPR license (Apache 2.0).

### CODE REVIEW

File: ppr-ui/src/components/registration/AmendmentDescription.vue [hunk 1]
```diff
@@ -28,7 +28,11 @@
     >
       <v-row class="pt-6">
         <v-col class="pa-4 pl-5 summary-text">
-          <p>
+          <p v-if="isSecurityActNotice">
```
Comment (cameron-eyds) [inline ppr-ui/src/components/registration/AmendmentDescription.vue:31]: Future state to expand on which PPR reg(s) have this copy, pending design review i think. Potentially all Crown Charge as well?

**Review by dimak1 — APPROVED** [review]
Looks great!

**Comment by bcregistry-sre** [comment]
Temporary Url for review: https://bcregistry-assets-dev--pr-1924-p1nj9j1m.web.app

File: ppr-ui/src/components/registration/AmendmentDescription.vue (hunk 1 above)
Comment (doug-lovett) [reply ppr-ui/src/components/registration/AmendmentDescription.vue:31]: All reg types can have a partial discharge if they have more than 1 active debtor. I would be surprised if Kevin is okay with this rule for other registration types.

Comment (cameron-eyds) [reply ppr-ui/src/components/registration/AmendmentDescription.vue:31]: Copy that, Sara had flagged this in designs with some potential future implementations. Will 100% defer to Kevin tho! For now, only SANs :)

**Review by doug-lovett — APPROVED** [review]
Looks good.

**Pull request merged**